### PR TITLE
Feat/events crud p2 6

### PR DESCRIPTION
**Pull request by Ahmed-Bensalem**

### CODE REVIEW

File: frontend/src/modules/Events/components/EventsList.jsx [hunk 1]
```diff
@@ -0,0 +1,349 @@
...
+
+  const fetchEvents = async () => {
+    try {
+      const response = await axiosRequest.get("/events");
+      setEvents(response.data);
+      setLoading(false);
+    } catch (error) {
+      console.error("Error fetching events:", error);
+      setLoading(false);
+    }
```
Comment (slimskhab) [inline frontend/src/modules/Events/components/EventsList.jsx]:
```suggestion
    try {
      const response = await axiosRequest.get("/events");
      setEvents(response.data);
    } catch (error) {
      console.error("Error fetching events:", error);
    }finally{
    setLoading(false);
    }
```

**Review by slimskhab — CHANGES_REQUESTED** [review]
1 change than LGTM

**Review by ALABM1 — APPROVED** [review]
LGTM

**Review by ALABM1 — APPROVED** [review]
LGTM

**Review by slimskhab — CHANGES_REQUESTED** [review]
- When I click on the event name it takes me to http://localhost:5173/app-academy-course-details.html
- Change the 34 to something live event Id or remove it
image
- I can choose a before date that is after the end date
image

**Review by slimskhab — APPROVED** [review]
LGTM!

**Pull request merged**